packages/theming/apple-theme/src/createAppleTheme.macos.ts

```diff
@@ -1,6 +1,27 @@
 import { ThemeReference } from '@fluentui-react-native/theme';
-import { defaultAppleThemeMacOS } from './appleTheme.macos';
+import { NativeModules } from 'react-native';
+import { BaseAppleThemeMacOS } from './appleTheme.macos';
 
 export function createAppleTheme(): ThemeReference {
-  return new ThemeReference(defaultAppleThemeMacOS);
+  const appleThemeReference: ThemeReference = new ThemeReference(BaseAppleThemeMacOS);
+
+  const module = NativeModules && NativeModules.MSFAppleThemeModule;
+  if (module) {
+    module.getApplePartialThemeWithCallback((error, applePartialTheme) => {
+      if (error) {
+        console.error(`Error retrieving apple theming module! ${error}`);
+      }
+      // Layer the native apple theming module values on top of the base Apple theme
+
+      // TODO this doesn't work. Why?
```

Comment:
Every TODO should be paired with a bug number, also any relevant info you have here would help explain it better to the next engineer who might look at it would be nice. e.g. why do we need this commented out line in the first place?

Reply:
The intention here was to add it before checkin in case I either found the solution, or confirmed with someone on CXE that this was an actual bug.

However, I think I figured out why this isn't working, and can probably remove this. There's a bunch of FluentTester code I'll have to change to fix it though :/

Reply:
I figured out what the problem was. It wasn't enough to update the themeReference with a partial theme. The details are weird but tl;dr I had to change the base/default theme for FluentTester on macOS to be the apple-theme. That's probably better anyway.

Reply:
Details for future reference:

A ThemeReference does not take other ThemeReferences as a way to layer themes on top of it, only partial themes or functions that return partial themes. This meant that even though I set up a callback and listener to load the partial theme from the native module on top of the base Apple theme in the createAppleTheme() ThemeReference, that didn't propagate up to the actual TesterThemeReference (what the Apple Theme was being applied on top of) getting invalidated and recalculated. That led to weird cases where the native theme wouldn't show up at all, or would only show up when the tester theme was invalidated for some other reason. 

The fix is to make the base theme of TesterThemeReference to the apple theme, and let the partial theme from the native module be a partial theme applied onto it.